waffle-hardhat/package.json

```diff
@@ -18,15 +18,17 @@
   "engines": {
     "node": ">=10.0"
   },
+  "dependencies": {
+    "ethers": "5.6.2",
+    "hardhat": "^2.9.2"
+  },
   "devDependencies": {
     "@ethereum-waffle/chai": "workspace:*",
     "@nomiclabs/hardhat-ethers": "^2.0.3",
     "@nomiclabs/hardhat-waffle": "^2.0.3",
```

Comment:
`@nomiclabs/hardhat-waffle` and `hardhat` - can only one of those be used?

Reply:
hardhat is peer dep of @nomiclabs/hardhat-waffle